src/characters.rs

```diff
@@ -3,14 +3,14 @@ use actix_web::{get, post, web, HttpResponse, Responder};
 use serde::{Deserialize, Serialize};
 use std::collections::HashMap;
 
-#[derive(Debug, Serialize)]
+#[derive(Debug, Serialize, PartialEq)]
 #[serde(rename_all = "camelCase")]
 pub struct Character {
-    name: String,
-    line_count: u32,
-    speaks_count: u32,
-    id: u32,
-    actor_id: Option<i32>,
+    pub name: String,
+    pub line_count: u32,
+    pub speaks_count: u32,
+    pub id: Option<u32>,
```

Comment:
Reusing this data structure made sense in javascript, but it could be worth having a separate structure for parsing so we can lean on rust's guarantee that id is always present when retrieving these from the database.